config/routes.rb

```diff
@@ -28,11 +28,20 @@
   resources :solr_documents, only: [:show], path: '/catalog', controller: 'catalog'
 
   mount Spotlight::Engine, at: '/'
+  concern :exportable, Blacklight::Routes::Exportable.new
 
   resources :exhibits, path: '/', only: [] do
     resource :dor_harvester, controller: :"dor_harvester", only: [:create, :update]
     resource :bibliography_resources, only: [:create, :update]
     resource :viewers, only: [:create, :edit, :update]
+
+    resources :solr_documents, except: [:index], path: '/catalog', controller: 'catalog' do
```

Comment:
Instead of overriding the normal routes we're getting from spotlight, would `only: []` work (instead of `except: [:index]`)?

Reply:
Yep! Added.. thanks